lib/gitx/cli/base_command.rb

```diff
@@ -12,7 +12,7 @@ class MergeError < Thor::Error; end
 
       add_runtime_options!
 
-      method_option :trace, :type => :boolean, :aliases => '-v'
+      method_option :trace, type: :boolean, aliases: '-v'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.